styleguide/derek/_partials/solutions/announcements.ejs

```diff
@@ -0,0 +1,53 @@
+<div class="container-fluid standard-padding bg-light-gray">
+  <div class="container">
```

Comment:
While I understand that the nested container issue is prevalent elsewhere this is a new design pattern and new markup that is free of past transgressions. I say address this issue in this pattern in this PR and leave the rest of the solutions pattern cleanup for another day.